### DISCUSSION TITLE
How to disable email verification for social networks (facebook)?

### DISCUSSION THREAD

**Discussion by Amarantine84** [ory/kratos#1712] [Q&A] [answered]
Hi!

In our projects we use registration via email/password and via social networks (facebook, vk, google).
When trying to register via social network (facebook, for example), user is asked to confirm registration via email. How to disable it for social network?

Thanks!

**Comment by vinckr** [comment]
Hello @Amarantine84 ,
sorry for the late reply!
The best way to solve this is probably to add some logic in the providers jsonnet to mark the email as verified when the account is created.
Most providers return an `email_verified` key: https://www.ory.sh/kratos/docs/guides/sign-in-with-github-google-facebook-linkedin#github

vk does not send this key, so the best practice would be to just send the verification email and verify the users email.

**Comment by zepatrik** [comment, marked as answer]
See this issue btw, it has all the info and current state of the feature https://github.com/ory/kratos/issues/1057
